kubernetes/loculus/values.schema.json

```
@@ -0,0 +1,998 @@
+            }
+          }
+        },
+        "initiallyVisible": {
+          "groups": ["metadata"],
+          "type": "boolean",
+          "description": "If true, the field is initially visible in the UI."
+        },
+        "hideOnSequenceDetailsPage": {
+          "groups": ["metadata"],
+          "type": "boolean",
+          "description": "If true, hides the field on the sequence details page."
+        },
+        "hideInSearchResultsTable": {
+          "groups": ["metadata"],
+          "type": "boolean",
+          "description": "If true, hides the field in the search results table (and makes it impossible to show)."
+        },
+        "perSegment": {
+          "groups": ["metadata"],
+          "type": "boolean",
+          "description": "Whether this is a metadata field that should exist for each segment. If so fields will be created as fieldName_A, fieldName_B in the case of an organism with segments A and B."
+        },
+        "customDisplay": {
+          "groups": ["metadata"],
+          "type": "object",
+          "description": "Custom display settings for the field on the sequence details page.",
+          "properties": {
+            "type": {
+              "groups": ["metadata"],
+              "type": "string"
+            },
+            "url": {
+              "groups": ["metadata"],
+              "type": "string"
+            }
+          }
+        },
+        "ontology_id": {
+          "groups": ["metadata"],
+          "type": "string"
+        },
+        "example": {
+          "groups": ["metadata"],
+          "type": "string",
+          "description": "Example value for the field."
+        },
+        "ingest": {
+          "groups": ["metadata"],
+          "type": "string",
+          "description": "Which NCBI field to map to this field."
+        },
+        "preprocessing": {
+          "groups": ["metadata"],
+          "type": "object",
+          "description": "The values of this field will be added to the preprocessing pipeline config file and the available values depend on the chosen pipeline. For the Nextclade pipeline, please see [here](https://github.com/loculus-project/loculus/blob/main/preprocessing/nextclade/README.md#custom-preprocessing-functions)."
+        },
+        "lineageSystem": {
+          "groups": ["metadata"],
+          "type": "string",
+          "description": "Use this on string fields that contain lineages, if you want to enable searches that can include sublineages. The value needs to be a lineage system that is defined under the `lineageSystemDefinitions` key."
+        }
+      },
+      "required": ["name"]
+    },
+    "organism": {
+      "type": "object",
+      "properties": {
+        "schema": {
+          "groups": ["organism"],
+          "docsIncludePrefix": false,
+          "type": "object",
+          "description": "Object of type [Schema](#schema-type)",
```

```suggestion
          "description": "Object of type [Schema](#schema-type)",
```

IMO this would be significantly clearer if called an `Organism Schema`

---

(or similar)

---

https://github.com/loculus-project/loculus/pull/3588/commits/3f8ca53e7ace5617a1bbf92e03c253b1d8ae47d0